editor/src/core/es-modules/package-manager/built-in-dependencies.ts

```diff
@@ -70,18 +80,21 @@ const BuiltInDependencies: Array<BuiltInDependency> = [
   ),
 ]
 
-function findBuiltInForName(moduleName: string): BuiltInDependency | undefined {
-  return BuiltInDependencies.find((builtIn) => builtIn.moduleName === moduleName)
+function findBuiltInForName(moduleName: string, isPreview: boolean): BuiltInDependency | undefined {
```

Comment:
I am a bit afraid of boolean flags like this, because where we are using it, you might miss what this means and the consequences. 

I personally like something like this much better: `mode: 'canvas' | 'preview'`. This means that when in the future we are copy pasting someone is more likely to notice that in the preview codebase this should not say 'canvas'